internal/config/config.go

```diff
@@ -98,31 +108,30 @@ type Samples struct {
 }
 
 type HighVolumeWrites struct {
-	Enabled bool     `yaml:"enabled"`
-	Samples Samples  `yaml:"samples"`
-	Readers *Readers `yaml:"readers,omitempty"`
-	Writers *Writers `yaml:"writers,omitempty"`
+	Enabled        bool            `yaml:"enabled"`
+	Configurations []Configuration `yaml:"configurations"`
 }
 
 type HighVolumeReads struct {
-	Enabled bool     `yaml:"enabled"`
-	Samples Samples  `yaml:"samples"`
-	Readers *Readers `yaml:"readers,omitempty"`
-	Writers *Writers `yaml:"writers,omitempty"`
+	Enabled bool `yaml:"enabled"`
+	Configuration
```

Comment:
Can we enable also `HighVolumeReads`, `HighVolumeAggregate` and `LogsBasedDashboard` to have multiple `Configurations`? For the sake of have a uniform format across scenarios.

Reply:
Yes, agree this is needed. Can we decide to do that in a following PR and not in this one. This PR is already beefy enought